internal/data/file_storage.go

```diff
@@ -0,0 +1,51 @@
+package data
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+)
+
+type FileStorage struct {
```

Comment:
With current implementation it looks like reader, not storage. Maybe rename this?

Or will it be extended later?

Reply:
renamed